assets/css/screen.css

```diff
@@ -152,90 +153,198 @@ body {
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    padding: 10vw 4vw;
+    z-index: 100;
+    padding: 6vw 3vw;
     min-height: 200px;
-    max-height: 450px;
-    text-align: center;
+    max-height: 340px;
 }
 
 .site-title {
     z-index: 10;
-    margin: 0;
+    margin: 0 0 0 -2px;
```

Comment:
to be checked: bigger size SF fonts is optically incorrect and looks like it's vertically misaligned with the smaller sized text. not sure this is needed as SF is only default for MacOS/iOS